manifests/istio-cni/templates/daemonset.yaml

```diff
@@ -78,6 +78,36 @@ spec:
               name: cni-bin-dir
             - mountPath: /host/etc/cni/net.d
               name: cni-net-dir
+        - name: repair-cni
+            {{- if contains "/" .Values.cni.image }}
+          image: "{{ .Values.cni.image }}"
+            {{- else }}
+          image: "{{ .Values.cni.hub | default .Values.global.hub }}/{{ .Values.cni.image | default "install-cni" }}:{{ .Values.cni.tag | default .Values.global.tag }}"
+            {{- end }}
+          imagePullPolicy: {{ .Values.cni.pullPolicy | default .Values.global.imagePullPolicy }}
```

Comment:
nit: this basically reverts https://github.com/istio/istio/pull/20707

Reply:
Added the wrapper from #20707